src/globals.d.ts

```diff
@@ -86,3 +86,11 @@ declare module 'es6-promisify' {
 declare module 'ethereumjs-abi' {
     const soliditySHA3: (argTypes: string[], args: any[]) => Buffer;
 }
+
+declare class HDWalletProvider {
+    constructor(mnemonic: string, rpcUrl: string);
+}
+
```

Comment:
Let's remove this space and add a comment above these two declarations similar to the other declarations in this file.